src/views/index.mustache

```diff
@@ -356,38 +356,38 @@
                     </button>
                 </div>
                 <div class="modal-body">
-                    <div class="help-title"><h4>Ways To Filter</h4></div><br>
+                    <div class="help-title"><h4>{{ways_to_filter}}</h4></div><br>
 
-                    <div class="help-title"><b>General Basics</b></div>
+                    <div class="help-title"><b>{{general_basics}}</b></div>
                     <div class="help-description">
                         <p>
-                            IV Range: <kbd>90-100</kbd> - 90-100% IV Pokemon<br>
-                            Level Range: <kbd>L30-35</kbd> - Level 30 to 35 Pokemon<br>
-                            Stats: <kbd>A0-1 & D14 & S15</kbd> - Pokemon with 0-1 Attack, 14 Defense & 15 Stamina<br>
-                            CP: <kbd>CP2000-3000</kbd> - All Pokemon with CP of 2000-3000<br>
-                            NOT: <kbd>!0-100</kbd> - All Pokemon that do not have their IVs scanned<br>
-                            PVP: <kbd>GL1-3 | UL1-3</kbd> - Pokemon that are rank 1-3 for either Great League or Ultra League<br><br>
+                            {{iv_range}}: <kbd>90-100</kbd> - {{iv_example}}<br>
```

Comment:
`iv_example` is a bit confusing as a name as it's not an example. What about `iv_example_help` or `iv_explanation` ? (same for others).